site/data/roadmap.yaml

```diff
@@ -1,43 +1,54 @@
-- item: Release 2.16
+- item: Release 2.17
   status: ⌚ Planned
-  year: 2024
-- item: Import PCAP files
+  year: 2025
+  sponsor: Checkmarx Ltd.
+  sponsorlink: https://checkmarx.com/
+- item: Improved Authentication Handling
+  status: ⚡ In progress
+  year: 2025
+  sponsor: Checkmarx Ltd.
+  sponsorlink: https://checkmarx.com/
+- item: Expand Custom Payloads support to additional rules
+  url: /docs/desktop/addons/custom-payloads/
+  status: ⌚ Planned
+  year: 2025
+- item: |
+    Import PCAP files - Iniial support has been released
```

Comment:
Typo in initial. Maybe this should be a Finished entry and this one changed to continue improving.